scripts/run

```diff
@@ -51,17 +54,24 @@ RUNTYPE=${1}
 
 run_tests() {
   cd "${ROOT_DIR}"
+  eval "$("${SCRIPTS_DIR}"/render-configs.sh -e)"
+
   # shellcheck disable=SC2086 # The array syntax and quoting seems to break gradle
   ./gradlew testUnit ${GRADLE_ARGS}
 
 #./gradlew --build-cache srcclr
 }
 
+wait_until_server_up() {
```

Comment:
Without this, integration tests would sometimes fail as the server wasn't up by the time the tests started running.